x-pack/plugins/index_lifecycle_management/__jest__/policy_table.test.tsx

```diff
@@ -181,8 +194,20 @@ describe('policy table', () => {
   test('displays policy properties', () => {
     const rendered = mountWithIntl(component);
     const firstRow = findTestSubject(rendered, 'policyTableRow').at(0).text();
```

Comment:
I think this data-test-subj now changed to `policyTableRow-${name}` so might need a little refactor in this part too